### PR TITLE
support swss vstest in python3

### PR DESCRIPTION
**Pull request by lguohan**
**What I did**
support swss vstest in python3

**Why I did it**
support python3

**How I verified it**
```
lgh@jenkins-worker-17:~/sonic-swss/tests$ sudo py.test -v | tee make.out                                                  ============================= test session starts ==============================
platform linux -- Python 3.8.2, pytest-4.6.2, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/lgh/sonic-swss/tests
plugins: flaky-3.6.1
collecting ... collected 257 items
============= 249 passed, 7 skipped, 1 xpassed in 6875.88 seconds ==============
```
**Details if related**

### CODE REVIEW

**Comment by lguohan** [comment]
retest vs please

**Comment by lguohan** [comment]
retest this please

File: tests/dvslib/dvs_acl.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def verify_acl_group_member(self, acl_group_ids, acl_table_id):
         member_groups = []
         for member in members:
             fvs = self.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP_MEMBER", member)
-            for k, v in fvs.items():
+            for k, v in list(fvs.items()):
```
Comment (daall) [inline tests/dvslib/dvs_acl.py:90]: Do you know why this change was necessary? I thought `items()` was compatible with python3.

File: tests/conftest.py [hunk 2]
```diff
@@ -136,17 +141,17 @@ def runcmd(self, cmd):
         try:
             out = subprocess.check_output("ip netns exec %s %s" % (self.nsname, cmd), stderr=subprocess.STDOUT, shell=True)
         except subprocess.CalledProcessError as e:
-            print "------rc={} for cmd: {}------".format(e.returncode, e.cmd)
-            print e.output.rstrip()
-            print "------"
+            print(("------rc={} for cmd: {}------".format(e.returncode, e.cmd)))
+            print((e.output.rstrip()))
```
Comment (daall) [inline tests/conftest.py:145]: Any idea why these extra parens are here?

File: tests/test_mirror.py [hunk 3]
```diff
@@ -235,7 +235,7 @@ def remove_fdb(self, vlan, mac):
 
     # Ignore testcase in Debian Jessie
     # TODO: Remove this skip if Jessie support is no longer needed
-    @pytest.mark.skipif(StrictVersion(platform.linux_distribution()[1]) <= StrictVersion('8.9'), reason="Debian 8.9 or before has no support")
+    @pytest.mark.skipif(StrictVersion(distro.linux_distribution()[1]) <= StrictVersion('8.9'), reason="Debian 8.9 or before has no support")
```
Comment (daall) [inline tests/test_mirror.py:238]: Is this line still necessary? (probably not in scope for this PR but thought I'd ask since it's here)

File: tests/dvslib/dvs_acl.py (hunk 1 above)
Comment (lguohan) [reply tests/dvslib/dvs_acl.py:90]: In Python 2, the methods items(), keys() and values() used to "take a snapshot" of the dictionary contents and return it as a list. It meant that if the dictionary changed while you were iterating over the list, the contents in the list would not change.

In Python 3, these methods return a view object whose contents change dynamically as the dictionary changes. Therefore, in order for the behavior of iterations over the result of these methods to remain consistent with previous versions, an additional call to list() has to be performed in Python 3 to "take a snapshot" of the view object contents.

Comment (lguohan) [reply tests/dvslib/dvs_acl.py]: remove list, keep it simple

File: tests/conftest.py (hunk 2 above)
Comment (lguohan) [reply tests/conftest.py]: it was added by 2to3. removed extra parens

File: tests/dvslib/dvs_acl.py (hunk 1 above)
Comment (daall) [reply tests/dvslib/dvs_acl.py]: thanks for the explanation, that makes a lot of sense!

**Comment by lguohan** [comment]
updated commit messages, @daall , need another signoff.
